### ISSUE TITLE
[css-fonts-4] font-weight:400 matching example is wrong

### ISSUE THREAD

**Issue by heycam** [w3c/csswg-drafts#3265]
https://drafts.csswg.org/css-fonts-4/#example-5babf945

This example says that if font B were eliminated, then the next font selected would be D.  But shouldn't it be C, since we should be checking for supported weights in the range [400, 500] in ascending order, and C is at 450?

**Comment by litherum** [comment]
Yes, you're right. There was a resolution a while ago that changed the search order, and I updated the graphs but must have forgotten to update the accompanying text, too.
